marklogic-data-hub-central/ui/src/components/record-view/record-view.tsx

```diff
@@ -0,0 +1,147 @@
+          </span> : emptyField}
+          {item.hubMetadata?.lastProcessedByStep ? <span className={styles.valText} data-testid={item.uri + '-lastProcessedByStep'}>
+            <OverflowTooltip title={item.hubMetadata.lastProcessedByStep} placement={"bottom"} content={item.hubMetadata.lastProcessedByStep} width={"200px"} />
+          </span> : emptyField}
+          {item.hubMetadata?.lastProcessedDateTime ? <span className={styles.valText} data-testid={item.uri + '-lastProcessedDateTime'}>
+            {CardViewDateConverter(item.hubMetadata?.lastProcessedDateTime)}
+          </span> : emptyField}
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <div id="record-data-card" aria-label="record-data-card" className={styles.rawDataCard}>
+      <Row gutter={24} type="flex" >
+        {props.data && props.data.length > 0 ? props.data.map((elem, index) => (
+          <Col key={index}>
+            <div >
+              <Card
+                className={styles.cardStyle}
+                size="small"
+              >
+                <div className={styles.cardMetadataContainer}>
+                  <span className={styles.uriContainer} data-testid={elem.uri + '-URI'}>URI: <span className={styles.uri}>
+                    <MLTooltip title={elem.uri} placement="bottom">{displayUri(elem.uri)}</MLTooltip></span></span>
+                  <span className={styles.cardIcons}>
+                    <MLPopover getPopupContainer={trigger => trigger.parentElement} content={displayRecordMetadata(elem)} placement="bottomRight" trigger="click">
+                      <span>
+                        <MLTooltip title={"View info"} placement="bottom">
+                          <span className={styles.infoIcon}><Icon type="info-circle" theme="filled" data-testid={elem.uri + '-InfoIcon'} /></span>
+                        </MLTooltip>
+                      </span>
+                    </MLPopover>
+                    <span className={styles.sourceFormat}
+                      style={sourceFormatStyle(elem.format)}
+                      data-testid={elem.uri + '-sourceFormat'}
+                    >{sourceFormatOptions[elem.format].label}</span>
+                    <MLTooltip title={'Detail view'} placement="bottom"
+                    ><i role="detail-link icon" data-testid={elem.uri + '-detailViewIcon'}><FontAwesomeIcon icon={faExternalLinkAlt} className={elem.format === 'binary' ? styles.detailLinkIconDisabled : styles.detailLinkIcon} size="lg" /></i>
+                    </MLTooltip>
+                  </span>
+                </div>
+                <div className={styles.snippetContainer} data-testid={elem.uri + '-snippet'} >
+                  {displaySnippet(elem)}
+                </div>
+              </Card>
+            </div>
+          </Col>)) : <span></span>}
+      </Row>
+    </div>
+  );
+}
+
+export default RawDataCardView;
```

Comment:
Can we rename it to RecordCardView?

Reply:
Oh was missing this change due to rebasing. Fixed. Thanks.